### DISCUSSION TITLE
HMVC

### DISCUSSION THREAD

**Discussion by butching** [ihsn/nada#50] [Q&A]
is this compatible with HMVC of codeigniter? if yes can you tell the procedure of integrating it. thanks

**Comment by mah0001** [comment]
No, HMVC is not supported. Are you thinking of adding new functionality to NADA?

**Reply by butching** [reply]
my company wants to add new functionality to NADA so im thinking of using HMVC to add that new module.

**Comment by mah0001** [comment]
You don't need HMVC to add new functionality. It would be useful if you could share some details on the new functionality.
